lighthouse-core/test/fraggle-rock/config/config-test.js

```diff
@@ -451,5 +462,4 @@ describe('Fraggle Rock Config', () => {
   });
 
   it.todo('should support plugins');
-  it.todo('should adjust default pass options for throttling method');
```

Comment:
this was just missed as part of https://github.com/GoogleChrome/lighthouse/pull/12873